client/src/services/mentorRegistry.ts

```diff
@@ -17,6 +17,31 @@ export interface MentorRegistry {
   updatedDate: Date;
 }
 
+export interface GetMentorRegistriesDto {
+  currentPage: number;
+  pageSize: number;
+  githubId?: string;
+  cityName?: string;
+  preferedCourses?: number[];
+  preselectedCourses?: number[];
+  technicalMentoring?: string[];
+}
+
+export interface GetMentorRegistriesResponse {
+  mentors: MentorRegistryDto[];
+  total: number;
+}
+
+export interface GetMentorRegistriesOptions {
+  currentPage?: number;
+  pageSize?: number;
+  githubId?: string;
+  cityName?: string;
+  preferedCourses?: number[];
+  preselectedCourses?: number[];
+  technicalMentoring?: string[];
+}
```

Comment:
DTO Adoption from Backend: Assuming you can directly use or extend DTOs defined by your backend, align your frontend interfaces with these DTOs. This approach reduces duplication and mismatches between the frontend and backend.
```
export interface GetMentorRegistriesOptions extends Partial<GetMentorRegistriesDto> {
}
```